oauth2client/contrib/gce.py

```diff
@@ -55,8 +52,7 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
```

Comment:
Sure. We didn't previously require that these credentials be refreshed as we never returned `service_account_email` as `None`. But that was seen as preferable to instantiating an `httplib2.Http` client as part of the `service_account_email` method.